README.md

```diff
@@ -3,68 +3,87 @@
 [![Build Status](https://travis-ci.org/cloudinary/cloudinary-vue.svg?branch=master)](https://travis-ci.org/cloudinary/cloudinary-vue)
 
 Cloudinary is a [cloud-based service](https://cloudinary.com/solutions) that provides an end-to-end image and video management solution
-including uploads,storage, manipulations, optimizations and delivery. 
+including uploads,storage, manipulations, optimizations and delivery.
 All your media resources are optimized and delivered through a fast CDN using industry best practices.
 
-Using Cloudinary's Vue.js SDK, you can easily perform smart image and video manipulations 
+Using Cloudinary's Vue.js SDK, you can easily perform smart image and video manipulations
 using code that integrates seamlessly with your existing Vue.js application, and deliver optimized and responsive media to your users.
 
-This Readme provides basic installation and usage information. For the complete documentation, see the [Vue.js SDK guide](https://cloudinary.com/documentation/vue_integration)_
+This Readme provides basic installation and usage information. For the complete documentation, see the [Vue.js SDK guide](https://cloudinary.com/documentation/vue_integration)\_
+
+## 🛠️ Installation
+
+### Vue 2.x
 
-# 🛠️ Installation 
-## Vue 2.x
 1. Install using Vue-CLI
-    - After you create your application with Vue-CLI, navigate to the created app folder, and install Cloudinary SDK by:
-        ```bash
-        vue add cloudinary
-        ```
-    - Set up your cloudName and pick the components to use (or use all 😃)
```

Comment:
did this actually change or is this is just formatting?

Reply:
just formatting. Reverted